libs/design-system/src/atoms/inputs/selectInputs.tsx

```diff
@@ -18,24 +21,13 @@ type SelectOptionsProps = {
 };
 
 export type SelectInputProps = {
-  /**
-   * The type of the input.
-   * It's 'primary' by default (big and white). "Secondary" is small and blue.
-   */
-  type?: 'primary' | 'secondary';
-
   /**
    * The id of the input.
    * It's also used as the name of the input.
    * It's also used as the label of the input.
    */
   id: string;
 
```

Comment:
Why removing `required`?

Reply:
I went too fast. I know that `required` is a default props on the native HTML select. So, since I removed that select for a custom one, I thought we could get rid of the required prop.

Now that you mention it though, I see how that's a mistake because we might still want to implement form validation and this prop could come in handy.

Will be rectified.